### ISSUE TITLE
Pagination: Previous/Next buttons don't announce the current page 

### ISSUE THREAD

**Issue by marvieqa** [CodeForPhilly/clean-and-green-philly#657]
## Description

When a screen reader user selects the Previous or Next button, it doesn't announce anything. Users have no context of what happened or what page they are in. We should have a mechanism to announce the current page when Previous or Next button is selected.

## Recommendation

- Add an `aria-live` region to announce the current page the Next of Previous button took them to.

**Comment by amyyeung17** [comment]
I would like to be assigned this :)

**Comment by bacitracin** [comment]
@amyyeung17 assigned!
